README.md

```diff
@@ -1,69 +1,110 @@
 # OctoTFS
 
-OctoTFS is a repository containing components for integration with Team Foundation Server and Visual Studio Team Services (VSTS).
+OctoTFS is a repository containing the Octopus Build and Release tasks for integration with Microsoft Team Foundation Server (TFS) and Azure DevOps (ADO) pipelines.  
 
-> Visual Studio Team Services (VSTS) was formerly known as Visual Studio Online (VSO)
+> Azure DevOps (ADO) was previously Visual Studio Team Services (VSTS) and previously Visual Studio Online (VSO)
 
-## Manual Building and Testing
+OctoTFS is made up of several tasks to make it easy to integrate TFS and ADO with Octopus Deploy. This is packaged up as an web extension that can be installed with TFS or ADO. 
 
-Note: Use the project in Octopus to deploy. These steps are only for manual releases, which should be avoided.
+## TFS / Azure DevOps Web Extension Custom Tasks 
 
-```
-sudo npm install -g tfx-cli
-# Increment version number from the one shown at https://marketplace.visualstudio.com/items?itemName=octopusdeploy.octopus-deploy-build-release-tasks-test&targetId=7b703d9c-2348-4d6d-b8fb-df60fdec5ec4&utm_source=vstsproduct&utm_medium=ExtHubManageList
-./pack.ps1 Test 2.0.96
-# Get access token from https://octopus-deploy.visualstudio.com/_details/security/tokens. Remember to select "All accessible accounts".
-./publish.ps1 Test 2.0.96 wieufvliuwefliquwefliqwevfliqwevfliqweuvfliqwevf
-```
+To learn more about how to use the extension and custom tasks, read the [VSTS README](source/vsts.md).
 
-## Team Build Preview Custom Steps
+* [Package Application](source/tasks/Pack)
+* [Push Package(s) to Octopus](source/tasks/Push)
+* [Create Octopus Release](source/tasks/CreateOctopusRelease)
+* [Deploy Octopus Release](source/tasks/Deploy)
+* [Promote Octopus Release](source/tasks/Promote)
 
-Custom Build Steps for [Team Build vNext](http://aka.ms/tfbuild)
+This extension provides a friendly interface to the [Octopus CLI](https://github.com/octopusdeploy/octopusclients) which does the heavy lifting to integrate.
```

Comment:
Should this use our shortlink `https://g.octopushq.com/ExternalToolOctoTools` instead?

Reply:
Yes, and I've raised OctopusDeploy/Octopurls#36 to update the shortlink with the new CLI source location.